package-overrides/github/angular/bower-material@0.9.0-rc1.json

```diff
@@ -0,0 +1,11 @@
+{
+  "main": "index",
```

Comment:
I thought about that... then disregarded. After consideration and your feedback, it does make sense to shim that one. Stand-by...